types/artifacts.d.ts

```diff
@@ -675,6 +677,12 @@ declare global {
         observedSpeedIndex: number;
         observedSpeedIndexTs: number;
       }
+
+      export interface Form {
+        attributes?: {id: string, name: string, autocomplete: string}
```

Comment:
let's add a comment to `attributes` that explains how if it is missing, this a formless set of elements.

Reply:
got it just added it @connorjclark